src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/ExtensibleSourceRepositoryProvider.cs

```diff
@@ -20,10 +20,14 @@ public sealed class ExtensibleSourceRepositoryProvider : ISourceRepositoryProvid
     {
 
         // TODO: add support for reloading sources when changes occur
-        private readonly Configuration.IPackageSourceProvider _packageSourceProvider;
+        private Configuration.IPackageSourceProvider _packageSourceProvider;
         private IEnumerable<Lazy<INuGetResourceProvider>> _resourceProviders;
         private Lazy<List<SourceRepository>> _repositories;
 
+        private readonly ISettings _settings;
```

Comment:
It won't make any difference, since ISettings is already MEF constrcuted even before coming here. I can still make it lazy just for the consistency